migrations/versions/de41bffc4830_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
+
+Revision ID: de41bffc4830
+Revises: c61a3c33adb5
+Create Date: 2017-06-20 11:04:16.344382
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'de41bffc4830'
+down_revision = 'c61a3c33adb5'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('event_invoices', sa.Column('invoice_pdf_url', sa.String(), nullable=True))
```

Comment:
Should be 
 `op.add_column('ticket_holders', sa.Column('pdf_url', sa.String(), nullable=False, server_default=''))`
i.e. it should contain some default value ( for already existing data ).
, cause if there is any single row of data already then it will contain **null** values in **pdf_url** column.

The error is : 
```
sqlalchemy.exc.IntegrityError: (psycopg2.IntegrityError) column "pdf_url" contains null values
 [SQL: 'ALTER TABLE ticket_holders ADD COLUMN pdf_url VARCHAR NOT NULL']
```